src/EnergyPlus/ReportSizingManager.cc

```diff
@@ -1696,8 +1696,6 @@ namespace ReportSizingManager {
 							}
 							SupFanNum = PrimaryAirSystem( CurSysNum ).SupFanNum;
```

Comment:
The problem is deeper than this.  When the defect file fails here, CurSysNum=3 and SupFanNum = 2.  That shouldn't be possible, because the defect file has four airloops with AirloopHVAC:UnitarySystem which all use Fan:SystemModel.  It also has one ZoneHVAC:UnitHeater which uses Fan:ConstantVolume. That's why SupFanNum=2 exceeds the array bounds on Fan.  Earlier calls here with SupFanNum=0 are fine, because FanDesHeat just sets the fan heat to zero and returns.

So, anyway, SupFanNum should be zero here.  Need to track down why it isn't.